suse_migration_services/units/prepare.py

```diff
@@ -159,6 +159,10 @@ def main():
                     '/var/lib/cloudregister'
                 ]
             )
+            log.info('Updating SMT cache')
+            Command.run(
+                ['/usr/sbin/updatesmtcache']
```

Comment:
ok you are calling this from inside of the live migration system. Does this really work ? where does the script come from ? Can we be sure this script exists in the live migration image ? ...and what exactly does it ?

Reply:
Yes, tested that change and migration succeeded. I can add if script exist check.

Reply:
The script is part of cloud-regionsrv-client. So as long as cloud-regionsrv-client is part of the image description the script will be there. However, a `if os.path.isfile('/usr/sbin/updatesmtcache')` is still prudent, IMHO

Reply:
ok cloud-regionsrv-client provides it and this package is part of the description, good

Reply:
Done.